capstone.css

```diff
@@ -445,89 +445,89 @@ footer {
 
 @media only screen and (max-width: 768px) {
   .Hamburger {
-  display: inline-block;
-  height: 20px;
-  width: 20px;
+    display: inline-block;
+    height: 20px;
+    width: 20px;
   }
 
   .firsthead {
-  padding-bottom: 0;
+    padding-bottom: 0;
   }
 
   .firsthead h1 {
-  font-size: 30px;
+    font-size: 30px;
   }
 
   .program {
-  height: 930px;
+    height: 930px;
```

Comment:
- [ ] On the mobile screens, the program cards are supposed to fill the space and align on top of one another. Please try to look at the comparison below 👇 
**Given design**
![image](https://user-images.githubusercontent.com/90222110/191800368-0d9333ba-7968-4e72-9ee5-c7a139022354.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/191800569-60f27ba5-35ce-49b9-ab10-77e1f18d67ed.png)

Reply:
Ok, I will try definitely.

Reply:
I am very sorry. I tried it but I am unable to do it.

Reply:
Please like the previous code reviewer asked please implement this change it should look like the design template you can achieve this by increasing the width of this container

Reply:
Ok. I am doing that. Thank you for your valuable opinion.

Reply:
It is done. I am attaching the screenshot.
https://user-images.githubusercontent.com/106825373/192056426-c07d76bd-5d49-4c16-aeb8-2b006cff6e7c.png